phpunit.xml

```diff
@@ -0,0 +1,20 @@
+<phpunit
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/7.0/phpunit.xsd"
+        colors="true"
+        backupGlobals="false"
+        bootstrap="vendor/autoload.php"
+>
+    <filter>
```

Comment:
Not really familiar with PHPUnit, but recent PHPUnit warnings seem to trigger a warning about this file:

```
Warning:       Your XML configuration validates against a deprecated schema.
Suggestion:    Migrate your XML configuration using "--migrate-configuration"!
```

Quick Google search suggests that it may be related to `<filter>` is removed/renamed in 9.3. https://github.com/laravel/dusk/pull/811

Is that something that requires thought, or is this good as is (for compatibility perhaps)?